src/Examples/45Example/Views/Account/Connect.cshtml

```diff
@@ -1,447 +1,95 @@
-                    </tr>
-                }
-            @if (Model.Nationality != null)
-            {
-                <tr>
-                    @{ var rowSpan = Model.Nationality.GetAnchors().Count() + 1;}
-                    <th rowspan="@rowSpan" scope="row">Nationality</th>
-                    <td rowspan="@rowSpan">
-                        @{
-                            string nationality = Model.Nationality.GetValue();
-                            @Html.DisplayFor(m => nationality);
-                        }
-                    </td>
-                    @{
-                        foreach (var anchor in Model.Nationality.GetAnchors())
-                        {
-                        <tr>
-                            <td>
-                                @{
-                                    string AnchorType = anchor.GetAnchorType().ToString();
-                                    @Html.DisplayFor(m => AnchorType);
-                                }
-                            </td>
-                            <td>
-                                @{
-                                    string AnchorValue = string.Join(" , ", anchor.GetValue());
-                                    @Html.DisplayFor(m => AnchorValue);
-                                }
-                            </td>
-                            <td>
-                                @{
-                                    string AnchorSubType = anchor.GetSubType().ToString();
-                                    @Html.DisplayFor(m => AnchorSubType);
-                                }
-                            </td>
-                        </tr>
-                    }
-                    }
-                    </tr>
-                }
-            @if (Model.StructuredPostalAddress != null)
-            {
-                <tr>
-                    @{ var rowSpan = Model.StructuredPostalAddress.GetAnchors().Count() + 1;}
-                    <th rowspan="@rowSpan" scope="row">Structured Postal Address</th>
-                    <td rowspan="@rowSpan"></td>
-                    @foreach (var anchor in Model.StructuredPostalAddress.GetAnchors())
-                    {
-                    <tr>
-                        <td rowspan="@Model.StructuredPostalAddress.GetValue().Count">
-                            @{
-                                string AnchorType = anchor.GetAnchorType().ToString();
-                                @Html.DisplayFor(m => AnchorType);
-                            }
-                        </td>
-                        <td rowspan="@Model.StructuredPostalAddress.GetValue().Count">
-                            @{
-                                string AnchorValue = string.Join(" , ", anchor.GetValue());
-                                @Html.DisplayFor(m => AnchorValue);
+        <section class="yoti-attributes-section">
+            <img class="yoti-company-logo" src="~/static/assets/company-logo.jpg" alt="company logo" />
```

Comment:
Should this show the real company logo if available? The PHP example is showing the static image too so perhaps this can be fixed in a separate task

Reply:
@KiranBali I don't suppose you know why we have the template logo and not the actual application profile's logo in the sdk-static-pages designs? Do you foresee any problems if we change it to use the logo if present, and if not then use this template logo? 
![image](https://user-images.githubusercontent.com/1381991/57839444-a22f2100-77be-11e9-840a-b5a17b47294e.png)

Reply:
I spoke to Kiran, and she doesn't see too much value in changing this - as it's just for a demo. We'll leave it as is